CHANGELOG.md

```diff
@@ -26,9 +26,11 @@
 ## Changes since v6.1.1
 
 - [#825](https://github.com/oauth2-proxy/oauth2-proxy/pull/825) Fix code coverage reporting on GitHub actions(@JoelSpeed)
+- [#767](https://github.com/oauth2-proxy/oauth2-proxy/pull/796) Deprecate GetUserName & GetEmailAdress for EnrichSessionState (@NickMeves)
```

Comment:
We have duplicate entries here by the looks of it

Reply:
Good catch - I even had the wrong issue # as the link text 😅 

I rebased and fixed it